includes/alerts.inc.php

```diff
@@ -36,7 +36,7 @@ function GenSQL($rule)
         return false;
     }
     //Pretty-print rule to dissect easier
-    $pretty = array('*'  => '*', '('  => ' ( ', ')'  => ' ) ', '/'  => '/', '&&' => ' && ', '||' => ' || ', 'DATE_SUB ( NOW (  )' => 'DATE_SUB( NOW()');
+    $pretty = array('*'  => '*', '('  => '(', ')'  => ')', '/'  => '/', '&&' => ' && ', '||' => ' || ', 'DATE_SUB ( NOW (  )' => 'DATE_SUB( NOW()');
```

Comment:
It looks like `'&&' => ' && '` and `'||' => ' || '` are the only two worthwhile mappings.  Probably just remove the rest.  It is interesting that this already contains two worthless replacements...